CHANGES.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## 1.11.0 (core) / 0.27.0 (libraries)
+
+## Major changes since 1.10.0 (core) / 0.26.0 (libraries)
+
+### Components — Configurable, reusable building blocks for data pipelines (now stable)
+
+Components, first released as Preview in 1.10.6, have reached Release Candidate status. APIs are stable, fully documented, and are the recommended, product-ready defaults for new Dagster projects.
```

Comment:
Should we ditch the "Release Candidate" language? I think the description after provides all the information necessary

Also: product-ready -> production-ready